pkg/cloud/services/powervs/service.go

```diff
@@ -18,6 +18,8 @@ package powervs
 
 import (
 	"context"
+	"fmt"
+	"github.com/IBM-Cloud/power-go-client/power/client/datacenters"
```

Comment:
Lets fix the import order like other files.

Reply:
Addressed